src/ui-kit/Search/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { setSearchWord } from 'redux/actions'
```

Comment:
redux is an absolute import, should not it be a separate line ? and prop-types and antd import on line 3 and 4 ?